packages/admin-ui/src/filters/filter-group.state.tsx

```diff
@@ -1,15 +1,17 @@
 import type { GenericFilterStateReturn } from './filter/filter.state'
 
 export function useFilterGroupState(props: UseFilterGroupStateReturn) {
-  const { filterStates } = props
+  const { filterStates, onClear: onClearCb = () => {} } = props
 
   const onClear = () => {
     filterStates.forEach((state) => state.onClear())
+    onClearCb()
```

Comment:
As the callback function is optional, you could use the optional chaining operator and remove the default value of the cb function:
```suggestion
    onClearCb?.()
```